datalad/log.py

```diff
@@ -270,16 +270,6 @@ def _clear_all(self):
             pb.clear()
 
 
-class NoProgressLog(logging.Filter):
```

Comment:
There only is a comment referring to it in l. 235 of this file (``(filtered away by NoProgressLog)``), maybe it should be removed, too, to avoid future confusion?